ch_backup/ch_backup.py

```diff
@@ -104,6 +106,22 @@ def list(self, state: BackupState = None) -> Sequence[BackupMetadata]:
 
         return backups
 
+    def _lock_id(self, operation: str) -> Optional[str]:
+        lock_id = operation
+        # if the replica name part of the id starts with '$' it means
+        # we used `platform.node()`
+        missing_shard = "/UNKNOWN"
+        missing_replica = f"/${platform.node()}"
+        lock_id = operation
+        try:
+            shard, replica = self._context.ch_ctl.get_shard_and_replica_name()
+            lock_id += f"/{shard}" if shard else missing_shard
+            lock_id += f"/{replica}" if replica else missing_replica
+        except ClickhouseError:
+            logging.error("Error in forming lock_id.", exc_info=True)
+            lock_id = f"{lock_id}{missing_shard}{missing_replica}"
```

Comment:
It could be not easy to change lock_id string uniformly in future - because of "try" and "except" different cases.
What do you think about:
```
try:
    shard, replica = self._context.ch_ctl.get_shard_and_replica_name()
except ClickhouseError:
    logging.error(...)

shard = shard or "UNKNOWN"
replica = replica or f"${platform.node()}"
  
return f"{lock_id}{shard}{replica}"
```
?